### ISSUE TITLE
Update leiningen dependencies

### ISSUE THREAD

**Issue by Saintis** [mtgred/netrunner#1943]
Several lein dependencies can be updates. Run [`lein ancient`](https://github.com/xsc/lein-ancient) to see which ones are "ancient":

```
[org.clojure/clojurescript "1.9.229"] is available but we use "1.7.228" (use :check-clojure to upgrade)
[org.clojure/core.async "0.2.385"] is available but we use "0.1.346.0-17112a-alpha"
[org.zeromq/jeromq "0.3.5"] is available but we use "0.3.4"
[cheshire "5.6.3"] is available but we use "5.4.0"
[org.omcljs/om "0.9.0"] is available but we use "0.8.8"
[sablono "0.7.4"] is available but we use "0.3.4"
[environ "1.1.0"] is available but we use "1.0.0"
[com.novemberain/monger "3.0.2"] is available but we use "3.0.0-rc2"
[org.slf4j/slf4j-nop "1.7.21"] is available but we use "1.7.12"
[differ "0.3.1"] is available but we use "0.2.1"
[figwheel "0.5.7"] is available but we use "0.5.2"
[figwheel-sidecar "0.5.7"] is available but we use "0.5.0-6"
```

Gave upgrading a quick try and some of these upgrades are breaking. Assuming these use syntactic versioning there should be some we can upgrade without any issues.

This can be auto-checked with one of those fancy GitHub badges by [Jarkeeper](https://jarkeeper.com/mtgred/netrunner).

**Comment by Saintis** [comment]
Updating `clojurescript`, `core.async` and `jeromq` locally. Does not appear to have caused any issues.

The `app.js` did increase about 20 KB though from what appears to be implementations of `indexOf` and `lastIndexOf` JS functions.

**Comment by zaroth** [comment]
@Saintis is this still valid for the `dev` branch or have they been updated already? If it's still outdated, feel free to update the dependencies and merge the changes there.

**Comment by Saintis** [comment]
Current status of `dev`

```
[org.clojure/clojurescript "1.9.229"] is available but we use "1.7.228" (use :check-clojure to upgrade)
[org.clojure/core.async "0.2.391"] is available but we use "0.1.346.0-17112a-alpha"
[org.zeromq/jeromq "0.3.6"] is available but we use "0.3.4"
[cheshire "5.6.3"] is available but we use "5.4.0"
[org.omcljs/om "0.9.0"] is available but we use "0.8.8"
[sablono "0.7.5"] is available but we use "0.3.4"
[environ "1.1.0"] is available but we use "1.0.0"
[com.novemberain/monger "3.1.0"] is available but we use "3.0.0-rc2"
[org.slf4j/slf4j-nop "1.7.21"] is available but we use "1.7.12"
[differ "0.3.1"] is available but we use "0.2.1"
[figwheel "0.5.8"] is available but we use "0.5.2"
[figwheel-sidecar "0.5.8"] is available but we use "0.5.0-6"
```

**Comment by JoelCFC25** [comment]
Can this be closed, or is the Sablono part reason enough to leave it open?
